android/Hybridge/project.properties

```diff
@@ -11,5 +11,6 @@
 #proguard.config=${sdk.dir}/tools/proguard/proguard-android.txt:proguard-project.txt
 
 # Project target.
-target=android-13
+target=android-21
 android.library=true
+android.library.reference.1=../../../mca-android-go/xwalk_core_library
```

Comment:
My local path is not same level based since I'm using that library in other projects (MCA), but in a last commit, prior to merge that should be fixed targeting a more comprehensible path like the one you claim. On the other hand of course, that dependency should be detailed and documented.